docs/content/installation/integrations/app-protect-waf-v5/configuration.md

```diff
@@ -0,0 +1,221 @@
+---
+docs: DOCS-000
+doctypes:
+- ''
+title: Configuration
+toc: true
+weight: 200
+---
+
+This document explains how to use F5 NGINX Ingress Controller to configure NGINX App Protect WAF version 5.
```

Comment:
```suggestion
This document explains how to use F5 NGINX Ingress Controller to configure [NGINX App Protect WAF version 5](link_to_waf_v5_docs).
```
I feel like we should have a link to the WAF v5 docs explicitly here since we're very clearly separating it from the v4 docs

Reply:
Made an edit suggestion for this.

Reply:
implemented